docs/guides/references/error-messages.mdx

```diff
@@ -805,8 +805,13 @@ documentation to learn more.
 The run you are attempting access to is already complete and will not accept new
 groups.
 
-When a run finishes all of its groups, it waits for a configurable set of time
-before finally completing. You must add more groups during that time period.
+When a run finishes all of its groups, it waits for a configurable set of time, [a run completion delay](/guides/cloud/smart-orchestration/parallelization#Run-completion-delay),
+before completing. You must run all `cypress run` calls with those more groups during that time period.
```

Comment:
I found the last sentence a little confusing. Are the "those more groups" referencing the ones that may have triggered this error initially? 

Maybe something like..."All `cypress run` calls with any new groups must be executed during that time period."

Reply:
Yah that makes sense. The time period starts when the last run finished with that same ciBuildId.

Reply:
Addressed in [`9e9f0f1` (#5391)](https://github.com/cypress-io/cypress-documentation/pull/5391/commits/9e9f0f16313b034173b7d1cf4697d5e754f32b07)